### PR TITLE
Update ICU State to County Distribution Weights Nov 2020

### PR DESCRIPTION
**Pull request by BrettBoval**
This PR updates the weights for State to County ICU using the past 30 days of data. To do that, I had to update the script to handle the new location_id key nomenclature.

For context, these re-weightings have been deliberately manual (instead of recalculated at runtime) to provide stability from the case where a single county reports an erroneous new case count (either due to backfill or fat finger). 

A more complex weighting system was considered (i.e. normalized temporal convolutions of previous 4 weeks cases for each date for each county in a state), but the explainability of the metric has been prioritized for now.

### Please Check
- [ ] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated?
- [ ] Are tests passing?

### CODE REVIEW

File: pyseir/icu/utils.py [hunk 1]
```diff
@@ -28,10 +28,13 @@ def calculate_case_based_weights() -> dict:
     last_month_cum_cases.name = "summed_cases"
 
     df = last_month_cum_cases.reset_index().dropna()
-    df["state_fips"] = df["fips"].str.slice(0, 2)  # Apply State Labels to Groupby
+    # Example location_id value = 'iso1:us#iso2:us-ak#fips:02013'
+    df["state_location_id"] = df["location_id"].str.split("#").str[1]  # 'iso2:us-ak'
```
Comment (TomGoBravo) [inline pyseir/icu/utils.py:32]: What do you want to happen for MSAs, that don't have a single state?

File: pyseir/icu/utils.py [hunk 2]
```diff
@@ -28,10 +28,13 @@ def calculate_case_based_weights() -> dict:
...
+    df["fips_location_id"] = df["location_id"].str.split("#").str[-1].str.split(":").str[-1]
```
Comment (TomGoBravo) [inline pyseir/icu/utils.py:33]: What depends on this value? It looks like the column name in `output` changed from `fips`, what about the reader?

Comment (TomGoBravo) [inline pyseir/icu/utils.py:33]: Is ` _add_fips_if_missing` at all handy? https://github.com/covid-projections/covid-data-model/blob/master/libs/datasets/timeseries.py#L328-L331

Comment (BrettBoval) [reply pyseir/icu/utils.py]: Updated to use that helper function instead of rolling my own parser.

Comment (BrettBoval) [reply pyseir/icu/utils.py]: Both "state_location_id" and "fips_location_id" are internal to this function. I switched to CommoFields.FIPS, but have to give more thought about generalizing this to move beyond the state-county mapping.

File: pyseir/icu/utils.py (hunk 1 above)
Comment (BrettBoval) [reply pyseir/icu/utils.py]: My first inclination is to sum up all the estimated ICU numbers from the county level. That breaks the hierarchy a little though. So you would disaggregate from states to counties and then reaggregate back to MSAs. There is an edge case where one state has actuals while the other has estimates that will result in a blended estimate, but I think that's the best we can do. Open to suggestions. Do you have any ideas?

**Review by ghop02 — APPROVED** [review]

**Pull request merged**